lighthouse.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 docker compose pull
+docker-compose -f docker-compose.yml -f docker-compose.ci.yml build
```

Comment:
Interesting, why do we need a build step now, and will we need to keep this going forward? tagging @alismx too.

Reply:
We need the build step here because the dockerfile has changed (to use node 18), without this it would use the image from main. 
I think we plan on keeping this in here and @alismx has a ticket [#4644] to try to build images only when needed.

Reply:
What @BobanL said! :smile: 

I'll add that If somebody wants to create a follow-up PR to this one to pull the build steps out after this goes in, that'd probably be fine, but it'd mean that we aren't covered in these types of cases until I get around to that ticket.

Reply:
thanks for explaining the plan, sounds good to me!